.github/workflows/flow-micro-benchmarks-runner.yml

```diff
@@ -0,0 +1,180 @@
+on:
+  workflow_call:
+    inputs:
+      architecture:
+        required: true
+        type: string
+      instance-type:
+        required: true
+        type: string
+      ami-id:
+        required: true
+        type: string
+
+env:
+  TAGS: | # Make sure there is no trailing comma!
+    [
+      {"Key": "Name",         "Value": "redisearch-ci-runner"},
+      {"Key": "Environment",  "Value": "CI"},
+      {"Key": "Run ID",       "Value": "${{ github.run_id }}"},
+      {"Key": "PR",           "Value": "${{ github.event.number }}"},
+      {"Key": "Owner",        "Value": "${{ github.actor }}"},
+      {"Key": "Project",      "Value": "${{ github.repository }}"}
+    ]
```

Comment:
Just so it's easy to distinguish between machines we launch on different occasions, even though I think this is the only place we have it in search today, let's add an indication its a micro benchmark instance